### ISSUE TITLE
Jaws does not read aria-labelledby with role=group

### ISSUE THREAD

**Issue by LauraOH** [FreedomScientific/VFO-standards-support#230]
### Summary

> Brief description of the issue. Please list any specific steps.

A div has a role of group with an accessible name provided by aria-labelledby. A nested field has n explicit label. When navigating by tabbing, jaws & edge is only providing the explicit field label; the grouping and group label is not provided.

### Expected result

Jaws should provide the aria-labelledby for the fields.

### Actual result

> What actually did happen?

Jaws just reads the element type (and associated explicit label, if it exists). 

### Example

```
 <div role="group" aria-labelledby="fieldlabel">
<label for="myfield">Explicit field label</label>
<BR>
<input type="text" id="myfield">
  </div>
<div id="fieldlabel">
  This is a field label as a div with an ID
</div>
```

#### JAWS version and build number
Jaws 2018.1811.30 

 
#### Operating System and version
Win OS 10

#### Browser and version:

MS Edge

**Comment by scottaohara** [comment]
@LauraOH  can you provide a test case, please?  Just to make sure we can reproduce the markup you're alluding to.

Thanks

**Comment by LauraOH** [comment]
Sorry I didn't markup the code as the proper format; corrected, also at 
https://codepen.io/LauraFO/full/gVRPzE

**Comment by scottaohara** [comment]
Thank you @LauraOH 

testing the codepen with JAWS 2019 + current Edge, i do see what you're referring to.  The group name is not announced when tabbing into the nameless text field.  Compared with JAWS 2019 + Firefox, Chrome and Chromium-Edge where the group's name IS announced.

Unfortunately, with the Edge browser about to be completely replaced by the new Chromium powered Edge, there are no plans for the JAWS developers to triage bugs related to the current browser.  

Thanks!
